### PR TITLE
Adding Geeetech A30 & FLSUN QQ-S machine definitons

### PR DESCRIPTION
**Pull request by curso007**
Dears,

I want to share my machine definitions of Geetech A30 & FLSUN QQ-S with the community.

Br,
Cataldo URSO.

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Developers, see issue CURA-6442.

**Comment by curso007** [comment]
> Developers, see issue CURA-6442.

Sorry, I do not find this CURA-6442 issue, or more likely I do not know how to find it! 
Can you post directly its link, sorry for the inconvenience.

**Comment by diegopradogesto** [comment]
@curso007 this is just for internal reference. Only the core Ultimaker team can access these tickets. 

We'll keep you posted about the progress. Thanks for the contribution.

File: resources/definitions/Geeetech_A30.def.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
+{
+    "id": "geeetech_a30",
+	"version": 2,
```
Comment (LipuFei) [inline resources/definitions/Geeetech_A30.def.json]: Please use 4 spaces instead TABs. The same for the other files.

File: resources/definitions/Geeetech_A30.def.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
+{
```
Comment (LipuFei) [inline resources/definitions/Geeetech_A30.def.json]: This file name needs to be the same as the ID, so it should be `geeetech_a30.def.json`

File: resources/definitions/Geeetech_A30.def.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    "id": "geeetech_a30",
+	"version": 2,
+	"name": "Geeetech A30",
+	"inherits": "fdmprinter",
+	"metadata": {
+		"author": "William & Cataldo URSO",
+		"manufacturer": "Shenzhen Geeetech Technology",
+		"setting_version": 1,
```
Comment (LipuFei) [inline resources/definitions/Geeetech_A30.def.json]: `setting_version` should be 8 now.

**Comment by patschi** [comment]
@curso007 Can you please let me know how you've found out that QQ-S is using "Repetier" G-Code flavor? Based on my knowledge they use the MKS Robin Mini board, which is according to https://github.com/makerbase-mks/MKS-Robin/tree/master/MKS%20Robin%20Mini/firmware using G-Code flavor of "Marlin".

**Comment by curso007** [comment]
Far I know, the orginal / official Flsun qq-s firmware is a variation of repetier firmware with closed source. The source code you are referencing to is quite recent and it not used yet by Flsun, and if you  look well there is not yet configuration for qq-s, at least the last time I looked.

**Pull request merged**